mmengine/visualization/vis_backend.py

```diff
@@ -340,6 +340,12 @@ class WandbVisBackend(BaseVisBackend):
             produced by the visualizer.
         init_kwargs (dict, optional): wandb initialization
             input parameters. Default to None.
+        define_metric_cfg (dict, optional):
+            A dict of metrics and summary for wandb.define_metric.
+            The key is metric and the value is summary.
+            When ``define_metric_cfg={'coco/bbox_mAP': 'max'}``,
+            The maximum value of``coco/bbox_mAP`` is logged on wandb UI.
+            Default: None
```

Comment:
can we refer to any documentation in wandb about the usages of `wandb.define_metric`. So that users can know all available summaries through the API doc.